runtime/src/lib.rs

```diff
@@ -237,7 +237,7 @@ impl frame_system::Config for Runtime {
 	/// A function that is invoked when an account has been determined to be dead.
 	///
 	/// All resources should be cleaned up associated with the given account.
-	type OnKilledAccount = ();
+	type OnKilledAccount = (); // TODO: Might implement something here?
```

Comment:
what do we want to cleanup when the account is killed ?

Reply:
Sorry. This was just a reminder for me, as I wanted to ask you if we want to do something there.

Reply:
ahh cool. We don't kill any module accounts so we can ignore it